omnipaxos/src/ballot_leader_election.rs

```diff
@@ -87,15 +87,13 @@ pub(crate) struct BallotLeaderElection {
     peers: Vec<NodeId>,
     /// The current round of the heartbeat cycle.
     hb_round: u32,
-    /// The heartbeat replies this instance received during the current round
+    /// The heartbeat replies this instance received during the current round.
     heartbeat_replies: Vec<HeartbeatReply>,
     /// The current ballot of this instance.
     current_ballot: Ballot,
-    /// The current leader of this instance
+    /// The current leader of this instance.
     leader: Ballot,
-    /// How this instance relates to its leader
-    election_status: ElectionStatus,
-    /// If this instance sees a need for a new leader
+    /// If this instance sees a need for a new leader.
```

Comment:
It would be nice to also document the implications of the value of happy. E.g., if we see any node that is happy, it means that there is a leader somewhere in the cluster that is making progress. Unhappy means that this specific node is seeking a new leader.